src/Template.php

```diff
@@ -210,14 +166,12 @@ public function displayBlock($name, array $context, array $blocks = [], $useBloc
      */
     public function renderParentBlock($name, array $context, array $blocks = [])
     {
-        if ($this->env->isDebug()) {
-            ob_start();
-        } else {
-            ob_start(function () { return ''; });
```

Comment:
You've removed this "feature". I'm fine as we won't be able to replicate it with yield, but that should be noted in the CHANGELOG.

Reply:
I've read https://github.com/twigphp/Twig/issues/1962 and https://github.com/twigphp/Twig/issues/3058 and I think this is fine now: both issues are handled by using yield instead of output buffering. I'd keep things as is.

Reply:
I updated the changelog to tell about what this PR does, but about this specific behavior, I can't figure out what to write. It's too complex to describe to really be a "feature".

Reply:
Maybe CoreExtension should keep that logic in its `captureOutput`. The issue of not leaking output buffers in case of fatal errors still exists when this BC layer is used.